numpyro/compat/handlers.py

```diff
@@ -0,0 +1,11 @@
+from jax import random
+
+from numpyro.handlers import *  # noqa: F401, F403
+from numpyro.handlers import seed as _seed
+
+
+# This is so that users do not have to import PRNGKey from jax.random.
+# XXX: Should we make this the default?
```

Comment:
I think we can hold a global `seed` (similar to _PYRO_STACK), which will change its value when `pyro.set_rng_seed(...)` is called. We will update this seed before feeding it into SVI.init or MCMC.run.

> Should we make this the default?

Sure. That would be convenient. Our current API does not require users use `random.split(...)`.